docs/cloud/index.md

```diff
@@ -33,6 +33,7 @@ For more resources on Azure AI, visit the cloud documentation site: [Build AI so
 azureai/quick-start
 azureai/manage-flows
 azureai/consume-connections-from-azure-ai
+azureai/create-run-with-automatic-runtime
```

Comment:
shall this be a subpage of 
![image](https://github.com/microsoft/promptflow/assets/47586720/df7385e3-fb6c-437b-b7a2-3962b069553b)

Reply:
![image](https://github.com/microsoft/promptflow/assets/7776147/a5faa0d3-b9cc-4776-b2b5-e53435cbca0a)
Updated